test/extended/builds/completiondeadlineseconds.go

```diff
@@ -33,24 +31,20 @@ var _ = g.Describe("[builds][Slow] builds should have deadlines", func() {
 			err := oc.Run("create").Args("-f", sourceFixture).Execute()
 			o.Expect(err).NotTo(o.HaveOccurred())
 
-			g.By("starting the build with --wait flag")
-			out, err := oc.Run("start-build").Args("source-build", "--wait").Output()
-			fmt.Fprintf(g.GinkgoWriter, "\nstart-build output:\n%s\n", out)
-			o.Expect(err).To(o.HaveOccurred())
+			g.By("starting the source build with --wait flag and short timeout")
+			br, err := exutil.StartBuildAndWait(oc, "source-build", "--wait")
```

Comment:
I believe that is true of the example you highlight. It tests whether the error affects the return code of "oc start-buid" when --wait is in use. Searching back through, I found a --wait that was redundant to --follow and a .By(...) string that mentioned --wait, but no longer used it. 
Both are now removed, but let me know if you disagree on the presence of others.